invoice/src/main/java/org/killbill/billing/invoice/dao/InvoiceDao.java

```diff
@@ -149,4 +149,32 @@ InvoicePaymentModelDao createRefund(UUID paymentId, BigDecimal amount, boolean i
      * @throws InvoiceApiException
      */
     void changeInvoiceStatus(UUID invoiceId, InvoiceStatus newState, InternalCallContext context) throws InvoiceApiException;
+
+    /**
+     * Save parent/child invoice relationship
+     *
+     * @param invoiceRelation the invoice relation object
+     * @param context the tenant context
+     * @throws InvoiceApiException
```

Comment:
Make sure to document (or remove) the Javadocs, otherwise the build may fail on jdk8 (doclint).